transformer_engine/pytorch/utils.py

```diff
@@ -251,11 +251,12 @@ def is_bf16_compatible() -> None:
     return torch.cuda.get_device_capability()[0] >= 8
 
 
-def non_tn_fp8_gemm_supported() -> bool:
+def is_non_tn_fp8_gemm_supported() -> bool:
```

Comment:
We could also call into that nvte function here, but I'm fine with leaving this in this PR.